src/shared/components/Category/component.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { CategoryWrapper, TitleWrapper, CountWrapper, IconWrapper } from './styles';
+
+type CategoryProps = {
+  title: string;
+  count?: string;
+  icon: string;
```

Comment:
we should add some `link` prop here, and pass it down, to make one category card to be a link.

Reply:
done